demo/admin/src/products/categories/ProductCategoryForm.gql.tsx

```diff
@@ -0,0 +1,40 @@
+// This file has been generated by comet admin-generator.
```

Comment:
should we remove this comments here? It's already scaffolded.

Reply:
Yes, missed that: 379121c8c2e88cc55fed61969922d36028149c72